tests/test.py

```diff
@@ -136,19 +137,31 @@ class TestSyncUtils(unittest.TestCase):
     def test_sync_blob(self):
         gcs_bucket_name, s3_bucket_name = os.environ["DSS_GCS_TEST_BUCKET"], os.environ["DSS_S3_TEST_BUCKET"]
         logger = logging.getLogger(__name__)
+        gcs_key_file = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
+        gcs = google.cloud.storage.Client.from_service_account_json(gcs_key_file)
         s3 = boto3.resource("s3")
-        payload = os.urandom(2**20)
+        payload, readback = os.urandom(2**20), b""
         test_key = "hca-dss-s3-to-gcs-sync-test"
+        try:
```

Comment:
would multiple runs of the same test not collide?  could you slap a unique suffix in the key?